com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/flowable/FlowableFacade.java

```diff
@@ -219,6 +246,24 @@ public void trigger(String executionId, Map<String, Object> variables) {
                      .trigger(executionId, variables);
     }
 
+    private boolean readyForDeletion(String processId) {
+        ProcessInstance processInstance = getProcessInstance(processId);
+        if (processInstance == null) {
+            LOGGER.info("Process " + processId + " has reached final state");
```

Comment:
I think it would be better to be on debug and to enable it explicitly on different landscapes?